packages/algob/sample-project/assets/fee-check.teal

```diff
@@ -1,4 +1,13 @@
+#pragma version 2
 // Check if the provided fee is at least 10,000 microalgos
```

Comment:
```suggestion
// Check if the provided fee is at most 10,000 microalgos
```
Right?

Reply:
I think this is good, this is just a sample example in which we require this specific condition

Reply:
Or it should be atleast 1000

Reply:
But why ? this depicts an example on how TEAL works and not how fees works ? right ?

Reply:
Ok. But i think this example directly contradicts teal guidelines.

Reply:
let's leave it.